_videos/fundamentals/create-a-new-module.md

```diff
@@ -98,6 +98,13 @@ The `registration.php` is a standardized file that follows the same pattern for
 
 The only thing that varies is the module name, which in our case is `Learning_FirstUnit`.
 
+## Run the "composer update" command
+
+After committing your changes and including your new module in the "require" section of the `composer.json` file, dump the new required autoloader file with the following command:
```

Comment:
@keithbentrup, what does _"dump the new required autoloader file"_ actually mean? We need to translate this into more plain language that uses less jargon.